app/controllers/browse_controller.rb

```diff
@@ -77,4 +78,10 @@ def changeset
   end
 
   def query; end
+
+  private
+
+  def require_moderator_for_unredacted_history
+    deny_access(nil) if params[:show_redactions] && !current_user&.moderator?
```

Comment:
I actually made `nil` the default for `deny_access` earlier today so if you rebase on master you can get rid of the argument here now.

Reply:
This change is not merged yet
https://github.com/openstreetmap/openstreetmap-website/pull/4540/files#diff-766c34fd6533171eaf54300c153f89d6002c35c02cfc9c5b219251f85180ad07

Reply:
Oh sorry about that.. I lost track of which things had and hadn't been merged over the weekend...